services/nginz/Makefile

```diff
@@ -114,8 +114,8 @@ zwagger-ui/swagger-ui: $(SWAGGER_BUNDLE)
 $(SWAGGER_BUNDLE):
 	curl -L https://github.com/swagger-api/swagger-ui/archive/v$(SWAGGER_VERSION).tar.gz -o $(SWAGGER_BUNDLE)
 
-src: $(NGINX_BUNDLE)
-	#Find keys on https://nginx.org/en/pgp_keys.html
+src: $(NGINX_BUNDLE) keys-downloaded
```

Comment:
Can't we make the rule depend on the GPG keyfile name? Then Make will automatically make sure it's not re-downloaded without us having to touch files manually?

Reply:
something like:

```suggestion
src: $(NGINX_BUNDLE)  $(NGINX_BUNDLE).asc

$(NGINX_BUNDLE).asc:
	./download-gpg-keys.sh
```